src/android/CameraManager.java

```diff
@@ -108,6 +111,14 @@ public void openDriver() throws IOException {
 			}
 			
 			if (DEBUG) Log.i(TAG, "Camera open success");
+
+			// Get the camera profile. Low quality is adequate for display our demo purposes right now.
+			camcorderProfile = CamcorderProfile.get(CamcorderProfile.QUALITY_LOW);
+
+			param = camera.getParameters();
+			param.setPreviewFrameRate(camcorderProfile.videoFrameRate);
+			camera.setParameters(param);
```

Comment:
Ideally, the quality could be passed as a parameter someplace, but that may be a more obtrusive change.